store/searchlayer/stop_word.go

```diff
@@ -0,0 +1,6 @@
+package searchlayer
+
+var (
```

Comment:
One question. Why do you wrap the variable instead of doing something like `var MYSQL...`?

Reply:
Maybe I am used to them, removing